### PR TITLE
update sqlite3, update lshw, add hwdata

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=lshw CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/hwdata.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  def self.build
+    system "./configure #{CREW_OPTIONS} \
+      --datadir=#{CREW_PREFIX}/share"
+    system 'make'
+  end
+
+  def self.install
+    system "make DESTDIR=#{CREW_DEST_DIR} install"
+    # Do not package blacklist of kernel modules
+    FileUtils.rm_rf CREW_DEST_LIB_PREFIX
```
Comment (uberhacker) [inline packages/hwdata.rb:37]: Why the need to remove all the libraries?

**Review by uberhacker — COMMENTED** [review]
The `LDFLAGS` for lshw are insane.

File: packages/hwdata.rb (hunk 1 above)
Comment (satmandu) [reply packages/hwdata.rb:37]: Addressed.

**Comment by satmandu** [comment]
> The `LDFLAGS` for lshw are insane.

Yup. Doesn't install otherwise.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**